includes/class-wc-query.php

```diff
@@ -443,19 +428,21 @@ public function get_catalog_ordering_args( $orderby = '', $order = '' ) {
 		$args    = array(
```

Comment:
`Warning: strtolower() expects parameter 1 to be string, array given in /srv/www/wordpress-default/public_html/wp-content/plugins/woocommerce/includes/class-wc-query.php on line 426` when sorting shop by average rating.

Reply:
`WordPress database error: [Unknown column 'wp_postmeta.meta_value' in 'order clause']
SELECT wp_posts.ID FROM wp_posts WHERE 1=1 AND wp_posts.post_type = 'post' AND ((wp_posts.post_status = 'publish')) ORDER BY wp_postmeta.meta_value+0 DESC, wp_posts.post_date DESC LIMIT 0, 5` in the sidebar for some reason when sorting by popularity